FirebaseAdmin/FirebaseAdmin/Messaging/AndroidConfig.cs

```diff
@@ -66,6 +66,12 @@ public sealed class AndroidConfig
         [JsonProperty("notification")]
         public AndroidNotification Notification { get; set; }
 
+        /// <summary>
+        /// Gets or sets the FCM options to be included in the message.
+        /// </summary>
+        [JsonProperty("fcm_options")]
+        public FcmOptions FcmOptions { get; set; }
```

Comment:
We need a new `AndroidFcmOptions` type here. See how this was implemented in Java: https://github.com/firebase/firebase-admin-java/pull/285